sparse/src/KokkosKernels_Handle.hpp

```diff
@@ -601,7 +601,36 @@ class KokkosKernelsHandle {
           "GS.");
     return cgs;
   }
+
+  // clang-format off
+  /**
+   * @brief Create a gauss seidel handle object
+   * 
+   * @param handle_exec_space The execution space instance to execute kernels on.
+   * @param num_streams The number of streams to allocate memory for.
+   * @param gs_algorithm Specifies which algorithm to use:
+   * 
+   *                     KokkosSpace::GS_DEFAULT  PointGaussSeidel
+   *                     KokkosSpace::GS_PERMUTED ??
```

Comment:
@brian-kelley or @lucbv: Can you help me complete in this documentation please?

Reply:
GS_DEFAULT is actually for both point and block. It turns into GS_PERMUTED for CPUs and GS_TEAM for GPUs. PERMUTED and TEAM are the same basic algorithm, the difference is that PERMUTED uses a RangePolicy over rows, but TEAM uses a TeamPolicy over batches of rows and ThreadVector parallelism within rows.

"PERMUTED" refers to the fact that rows/columns of the matrix are reordered into colors for better locality.

Reply:
Thanks, @brian-kelley. I'll update these docs in a follow-on PR.

@brian-kelley, @lucbv: I've addressed all your feedback. Are you ready to merge?